pkg/git/message.go

```diff
@@ -19,7 +19,7 @@ type Message struct {
 	Headers map[string]string
 }
 
-var headerRe = regexp.MustCompile("^([^:]+):([^:]+)$")
+var headerRe = regexp.MustCompile("^([^:]+):([^:/]+)$")
```

Comment:
I am thinking where the regexp should be more restrictive. And I think it should be on the header name side rather than the value.
Header names should be letters and `-` while values, including for headers should be anything 

```suggestion
var headerRe = regexp.MustCompile("^([a-zA-z0-9-]+)[ ]*:(.+)$")
```